### PR TITLE
SettingsCard: Fix links display

### LINKED ISSUE #1965: Links on settings screen have confusing design
On the main settings screen, our links for the terms of service and privacy policy extend all the way to the left of the screen, much wider than the actual text labels.

This is confusing because what looks like empty space is actually part of the link, and tapping on it opens up the ToS or privacy policy page.

[Above reframed by @gnprice. Original report below:]

----

screenshot_20180224-143955__01

Clicking in the highlighted area also produces a click effect. This can be fixed along side #1959

### PR DESCRIPTION
**Pull request by xamfy**
screenshot_20180306-150350
Fixes #1965

### CODE REVIEW

**Comment by xamfy** [comment]
okay, I'll try

**Comment by xamfy** [comment]
@borisyankov this works

File: src/settings/SettingsCard.js [hunk 1]
```diff
@@ -24,6 +24,10 @@ const styles = StyleSheet.create({
     alignItems: 'flex-end',
     flexDirection: 'row',
   },
+  link_container: {
```
Comment (jainkuniya) [inline src/settings/SettingsCard.js]: `linkContainer` will be better name for this one.

**Comment by xamfy** [comment]
@jainkuniya done

File: src/styles/theme.js [hunk 2]
```diff
@@ -93,6 +93,7 @@ export default ({ color, backgroundColor, borderColor, cardColor }: Props) => ({
     fontSize: 15,
     color: BRAND_COLOR,
     textAlign: 'right',
+    flex: 0.5,
```
Comment (borisyankov) [inline src/styles/theme.js]: That means roughly 'half the width of other elements'
This is unlikely to be a good idea for a general-purpose style that will be used in different places.

Comment (xamfy) [reply src/styles/theme.js]: got it, but can I use ```flex: 0.5```  just in the SettingsCard, instead of setting a width

**Comment by gnprice** [comment]
This has been idle for a few weeks, and I think it won't end up leading to a change we merge. Closing.

@xamfy , thanks for your contribution, and I hope you'll look around for another issue that's a better match!

**Pull request closed without merging**